src/stores/VideoChannelStore.ts

```diff
@@ -172,8 +177,9 @@ export default class VideoChannelStore extends AsyncStoreWithClient<null> {
             },
         );
         messaging.transport.send(ElementWidgetActions.JoinCall, {
-            audioDevice: audioDevice?.label,
-            videoDevice: videoDevice?.label,
+            audioOutput: audioOutput?.label,
+            audioInput: audioInput?.label,
+            videoInput: videoInput?.label,
```

Comment:
I really have no idea how this code works but I am wondering why is `label` used here rather than `id`?

Reply:
Jitsi's API requires the label to be passed, for some reason